readdy_documentation/_home/get_started.md

```diff
@@ -29,8 +29,8 @@ simulation.run(100, 0.01)
 ```
 
 The above snippet performs a ReaDDy simulation, which consists of three steps:
-1. [Configure the system]({{site.baseurl}}/system_conf.html)
-2. [Setup the simulation]({{site.baseurl}}/simulation_setup.html)
-3. [Run and analyze results]({{site.baseurl}}/run_and_results.html)
+1. [Configure the system]({{site.baseurl}}/system.html)
+2. [Setup the simulation]({{site.baseurl}}/simulation.html)
```

Comment:
maybe include the `run` here in the second step, and have `post processing` in the third, as the navigation now suggests